nullaway/src/test/java/com/uber/nullaway/NullAwayCoreTests.java

```diff
@@ -556,6 +556,39 @@ public void testMapPutAndPutIfAbsent() {
         .doTest();
   }
 
+  @Test
+  public void testMapComputeIfAbsent() {
+    defaultCompilationHelper
+        .addSourceLines(
+            "Test.java",
+            "package com.uber;",
+            "import java.util.Map;",
+            "import java.util.function.Function;",
+            // Need JSpecify (vs javax) for annotating generics
+            "import org.jspecify.nullness.Nullable;",
+            "class Test {",
+            "   Object testComputeIfAbsent(String key, Function<String, Object> f, Map<String, Object> m){",
+            "     m.computeIfAbsent(key, f);",
+            "     return m.get(key);",
+            "   }",
+            "   Object testComputeIfAbsentLambda(String key, Map<String, Object> m){",
+            "     m.computeIfAbsent(key, k -> k);",
+            "     return m.get(key);",
+            "   }",
+            "   Object testComputeIfAbsentNull(String key, Function<String, @Nullable Object> f, Map<String, Object> m){",
+            "     m.computeIfAbsent(key, f);",
+            "     // BUG: Diagnostic contains: returning @Nullable expression from method with @NonNull return type",
+            "     return m.get(key);",
+            "   }",
+            "   // ToDo: should error somewhere, but doesn't, due to limited checking of generics",
```

Comment:
This worries me a bit, but the alternative if that `m.computeIfAbsent(...)` usually has to be followed by a `castToNonNull(m.get(...)))`

Reply:
Yeah, I agree that the most common case will be that `computeIfAbsent()` returns a non-null result.  So this soundness tradeoff seems right, until we do better with generics